Source/Charts/Utils/Platform+Accessibility.swift

```diff
@@ -33,7 +33,7 @@ open class NSUIAccessibilityElement: UIAccessibilityElement
         }
     }
 
-    override init(accessibilityContainer container: Any)
```

Comment:
@jjatie not familiar with Accessibility. Is it best practice to make it public? If so, why the super class didn't do that...